portfolio/src/main/webapp/script.js

```diff
@@ -47,17 +47,55 @@ function newCommentBox(comment){
   return commentBox;
 }
 
+function clearChildren(dom) {
+  while(dom.lastElementChild) {
+    dom.removeChild(dom.lastElementChild);
+  }
+}
+
 /**
- * Adds a greeting to the page using Promises
+ * Adds comments to the page using Promises
  */
 function loadComments() {
-  fetch('/data').then(response => response.json()).then((comments) => {
+  const commentContainer = document.getElementById('comment-container');
+  const parameters = location.search;
+  clearChildren(commentContainer);
+
+  // Repopulate comment section
+  fetch('/data'+parameters).then(response => response.json()).then((comments) => {
     for(const comment of comments) {
-      document.getElementById('comment-container').appendChild(newCommentBox(comment));
+      commentContainer.appendChild(newCommentBox(comment));
     }
   });
 }
 
+function verifyDelete() {
+  var inputWord = prompt('What\'s the magic word?');
+  if (inputWord == 'please') {
+    alert('Comments have been deleted.')
+    return true;
+  } else {
+    alert('That word is not magic.');
+    return false;
+  }
+}
```

Comment:
lol

Nit: I shared with all of you one of my favorite ToTTs. It basically talks how the less indentation you have the better. In this case you don't need an else block because you return inside the if block.